src/controllers/IllnessEndDateController.ts

```diff
@@ -1,9 +1,9 @@
 import { SessionMiddleware } from 'ch-node-session-handler';
 import { controller } from 'inversify-express-utils';
 import moment from 'moment';
+import { IllnessEndDateValidator } from './validators/IllnessEndDateValidator';
```

Comment:
For consistency better use the src application path 
`import { IllnessEndDateValidator } from 'app/controllers/validators/IllnessEndDateValidator';`

Reply:
Ah yes, ok 👍